### ISSUE TITLE
MediaQuery: use "where" clause to exclude nextcloud folder

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#1397]
Try to use something like "not like '%nextcloud%'" to exclude nextcloud folders while scanning for avilable auto upload folders. This should be better than the current implementation where we get a list of all possible images and filter them afterwards.

As discussed with @AndyScherzinger on the train ;-)

**Comment by mario** [comment]
What if I have a folder called nextcloud_123? That would exclude that too, and it shouldn't..

**Comment by AndyScherzinger** [comment]
Right, the idea would be to exclude the installation/storage path(s) of Nextcloud on the device. So yeah, the clause would have to be slightly more precise than just %nextcloud%.

**Comment by tobiasKaminsky** [comment]
Can we use MainApp.getStoragePath() for this? This should exclude all account folders.
If so, then I would prepare a PR for this.

**Comment by mario** [comment]
No, because this is broken. See https://github.com/nextcloud/android/issues/1707.

**Comment by joshtrichards** [comment]
Should be less broken now since #1707 has been addressed.
